.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     needs: code_quality
     strategy:
       matrix:
-        ckan-version: ['2.10', '2.9']
+        ckan-version: ['2.11', '2.10', '2.9']
```

Comment:
Since we already use the ckan docker container for the major version
``image: ckan/ckan-dev:${{ matrix.ckan-version }}`` https://hub.docker.com/r/ckan/ckan-dev
https://github.com/ckan/ckan-docker#5-ckan-images
which is at the head of the 'released' aspects.

we could swap it to just keep it vanilla unless it sees that we set the ckan-git* options in the matrix extras
i.e

```
continue-on-error: ${{ matrix.experimental }}
strategy
  max-parallel: 4 #only 4 jobs at once
  matrix:
        ckan-version: ['2.11', '2.10', '2.9'] 
        experimental: [false]
        include:
          - ckan-version: '2.9'
            ckan-git-version: 'ckan-2.9.9-qgov.4'
            ckan-git-org: 'qld-gov-au'
            experimental: true
          - ckan-version: '2.9'
            ckan-git-version: 'qgov-master-2.9.9'
            ckan-git-org: 'qld-gov-au'
            experimental: true
          - ckan-git-org: 'ckan'
            ckan-git-version: 'master'
            experimental: true   #master is unstable, good to know if we are compatible or not

also add ability to skip if master fails (as its a heads up)

```

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow

Reply:
I can try some of this. Not sure why our fork would be considered experimental though?